vllm/model_executor/layers/mamba/mamba_mixer.py

```diff
@@ -60,9 +61,13 @@ def __init__(self,
         # doesn't allow to override it
         self.conv1d.weight.data = self.conv1d.weight.data.unsqueeze(1)
 
-        self.in_proj = MergedColumnParallelLinear(hidden_size,
-                                                  [intermediate_size] * 2,
-                                                  bias=use_bias)
+        self.in_proj_lin = ColumnParallelLinear(hidden_size,
```

Comment:
QQ: Why modify this?

Reply:
the LoRA kernels did not work for us with MergedColumnParallelLinear so I split this param into two ColumnParallelLinear which are supported. This modification also required changes to our weight loading methods.

Reply:
This layer has 2 loras, and the slicing logic for each lora is different from llama's, right?

Reply:
Yes, the reason is that for llama the gate_up_proj is a stacked param and in Jamba/Mamba in_proj is not.

Reply:
Just add the mapping for `in_proj` as shown below, there's no need to modify the loading logic

 packed_modules_mapping = {
        "qkv_proj": [
            "q_proj",
            "k_proj",
            "v_proj",
        ],
        "in_proj":["in_proj"]
    }